Sources/FoundationEssentials/FileManager/FileOperations.swift

```diff
@@ -553,13 +553,16 @@ enum _FileOperations {
                 try source.withNTPathRepresentation { pwszSource in
                     var faSourceAttributes: WIN32_FILE_ATTRIBUTE_DATA = .init()
                     guard GetFileAttributesExW(pwszSource, GetFileExInfoStandard, &faSourceAttributes) else {
-                        throw CocoaError.moveFileError(GetLastError(), src, dst)
```

Comment:
To clarify, I think the unit test should use `shouldProceedAfterMoveError` and ensure it has a callback in the list, instead of checking that `shouldProceedAfterCopyError` (which is unrelated) is empty. That's what `testRemoveItemAtPath` does

Reply:
Done

Reply:
I think we need to update the unit test to list the path in the array (to fix the failures on macOS/Darwin), which will then cause it to fail on Windows and we should call `_shouldProceedAfter(error:)` here to correct that failure